ci-operator/config/openshift/jenkins/master.json

```diff
@@ -0,0 +1,172 @@
+{
+    "tag_specification": {
+        "cluster": "https://api.ci.openshift.org",
+        "namespace": "openshift",
+        "name": "jenkins",
```

Comment:
I'm still unsure about this step's specification ... especially ptal here

Reply:
What are you trying to do when your image is "promoted"?  Should it be in "openshift/origin-jenkins:v3.11.0"?  If so, this section needs to look like all the others.

Reply:
"promotion" today for jenkins is pushing the centos image to docker.io/openshift/jenkins-2-centos7:v3.11

doesn't seem to map to what tag_specification is doing

and excluding tag_specification fails ci-operator's dependency check